Dockerfile

```diff
@@ -11,13 +11,13 @@ COPY ./package.json package.json
 COPY ./package-lock.json package-lock.json
 COPY ./prisma/schema.prisma prisma/schema.prisma
 
-RUN apt update && apt install -y \
-  g++ \
-  git \
-  make \
-  openssl \
-  python3 \
-  && rm -rf /var/lib/apt/lists/*
+RUN apt-get update && apt-get install -y --no-install-suggests \
```

Comment:
Why changing `apt` to `apt-get`?

Reply:
@dtslvr I've replaced `apt` with `apt-get` due to the following lint warning: https://github.com/hadolint/hadolint/wiki/DL3027

Reply:
Okay! 👍🏻